core/DependencyInjection/ContainerBuilder.php

```diff
@@ -165,9 +175,16 @@ private function getTemporaryServiceFileContent(): string
         return vsprintf(
             '<?php
         use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+        use \oat\generis\model\Middleware\MiddlewareExtensionsMapper;
         
         return function (ContainerConfigurator $configurator): void
         {
+            $parameter = $configurator->parameters();
+            $parameter->set(
+                MiddlewareExtensionsMapper::MAP_KEY, 
+                ' . var_export($this->middlewareExtensionsMapper->map($extensions), true) . '
```

Comment:
Is it possible to use `%s` + adding this value to `$contents` ?

Reply:
agreed. Added